tests/python/pants_test/backend/jvm/tasks/jvm_compile/java/test_cache_compile_integration.py

```diff
@@ -21,6 +21,12 @@ class Compile(namedtuple('Compile', ['srcfiles', 'config', 'artifact_count'])):
 
 class CacheCompileIntegrationTest(BaseCompileIT):
 
+  @classmethod
+  def should_configure_pantsd(cls):
+    """TODO: See the point about watchman in #7320.
```

Comment:
TODOs are better in the form `TODO(7320)`.

Also docstring can be consolidated into one line by moving the `"""` up to this same line.